shared/macros-highlevel.jinja

```diff
@@ -1,15 +1,18 @@
 {{% macro ocil_package(package) -%}}
+  {{% if pkg_system is defined %}}
```

Comment:
I might know the answer, but could you clarify for which products `pkg_system` would be undefined?

Reply:
Any product that doesn't have one. Basically, macros should not be required to be used in the content.